static/styles/portico/portico_signin.css

```diff
@@ -401,6 +401,33 @@ html {
         font-weight: 400;
     }
 
+    .demo-org-warning {
```

Comment:
Just a note that we probably want to refer to these using more consistent terminology, in the variables and CSS; `demo-realm` or `demo-organization` seems clearly better than `demo-org`.